### DISCUSSION TITLE
Open Alpha on last buffer close

### DISCUSSION THREAD

**Discussion by BaDxKaRMa** [LazyVim/LazyVim#1150] [Q&A] [answered]
I recently migrated from a custom config over to LazyVim. Everything seems to be working great except for one thing. I had this autocmd to auto launch Alpha when I would :bd close my last buffer, but I can't seem to make it happen. This is in my autocmds.lua. Other autocmds are working, but not this one.

```lua
vim.api.nvim_create_augroup("alpha_on_empty", { clear = true })
vim.api.nvim_create_autocmd("User", {
  pattern = "BDeletePost*",
  group = "alpha_on_empty",
  callback = function(event)
    local fallback_name = vim.api.nvim_buf_get_name(event.buf)
    local fallback_ft = vim.api.nvim_buf_get_option(event.buf, "filetype")
    local fallback_on_empty = fallback_name == "" and fallback_ft == ""

    if fallback_on_empty then
      vim.cmd("Alpha")
    end
  end,
})
```

**Comment by jyuan0** [comment, marked as answer]
Did you replace [mini.bufremove](https://github.com/echasnovski/mini.bufremove) with [bufdelete.nvim](https://github.com/famiu/bufdelete.nvim)? LazyVim uses mini.bufremove by default. `BDeletePost` is a bufdelete.nvim event.

**Reply by BaDxKaRMa** [reply]
Thanks so much. This was my problem. After swapping plugins, everything is as expected.
